pkg/webhook/bootstrap.go

```diff
@@ -255,18 +268,26 @@ func (s *Server) clientConfig() (*webhookClientConfig, error) {
 	return cc, nil
 }
 
-func (w *webhookClientConfig) withPath(path string) error {
-	if w.URL != nil {
-		u, err := url.Parse(*w.URL)
+func (s *Server) clientConfigWithPath(path string) (*admissionregistration.WebhookClientConfig, error) {
+	cc, err := s.clientConfig()
+	if err != nil {
+		return nil, err
+	}
+	return cc, setPath(cc, path)
+}
+
+func setPath(cc *admissionregistration.WebhookClientConfig, path string) error {
+	if cc.URL != nil {
+		u, err := url.Parse(*cc.URL)
 		if err != nil {
 			return err
 		}
 		u.Path = path
 		urlString := u.String()
-		w.URL = &urlString
+		cc.URL = &urlString
 	}
-	if w.Service != nil {
-		w.Service.Path = &path
+	if cc.Service != nil {
+		cc.Service.Path = &path
 	}
 	return nil
 }
```

Comment:
hmm..... may be we need `ccWithPath type` with methods it ?

Reply:
Made some changes here.